### PR TITLE
test: TTU test for integration

### PR DESCRIPTION
**Pull request by adriantam**
## Description
First set of TTU test cases for integration

## Review Checklist
- [ ] I have clicked on ["allow edits by maintainers"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [ ] I have added documentation for new/changed functionality in this PR or in a PR to [openfga.dev](https://github.com/openfga/openfga.dev) [Provide a link to any relevant PRs in the references section above]
- [ ] The correct base branch is being used, if not `main`
- [ ] I have added tests to validate that the change in functionality is working as expected

### CODE REVIEW

File: tests/check/check_ttu.go [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+			{
+				Name:        "invalid_user",
+				Tuple:       &openfgav1.TupleKey{Object: "ttus:1", Relation: "direct_pa_direct_ch", User: "user:invalid"},
+				Expectation: false,
+			},
+			{
+				Name:        "invalid_group",
+				Tuple:       &openfgav1.TupleKey{Object: "ttus:1", Relation: "direct_pa_direct_ch", User: "user:invalid_group"},
+				Expectation: false,
+			},
```
Comment (miparnisari) [inline tests/check/check_ttu.go:33]: Nitpick: unsure what value this test is adding, it looks identical to the one above it

File: tests/check/check_ttu.go [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+	{
+		Name: "userset_pa_userset_ch",
+		Tuples: []*openfgav1.TupleKey{
+			{Object: "ttus:5", Relation: "userset_parent", User: "usersets-user:ttu_5"},
+			{Object: "usersets-user:ttu_5", Relation: "userset", User: "directs-user:ttu_5#direct"},
+			{Object: "directs-user:ttu_5", Relation: "direct", User: "user:valid"},
+			{Object: "directs-user:ttu_5_unconnected", Relation: "direct", User: "user:ttu_5_unconnected_directuser"},
+			{Object: "directs-user:ttu_5_unconnected_userset", Relation: "direct", User: "user:ttu_5_unconnected_userset"},
+			{Object: "usersets-user:ttu_5_unconnected_userset", Relation: "userset", User: "directs-user:ttu_5_unconnected_userset#direct"},
+		},
```
Comment (miparnisari) [inline tests/check/check_ttu.go:157]: Could we also add assertions for the case where an object has a parent (ie similar to line 151) and the parent has an associated userset, but that userset has no members so we expect `allowed=false` for any input user?

File: tests/check/check_ttu.go (hunk 1 above)
Comment (adriantam) [reply tests/check/check_ttu.go:33]: Will change to ttu_invalid_group.  Want to test the case where there is such user belong to `directs-user`, but that `directs-user` does not connect to the TTU itself.

File: tests/check/check_ttu.go (hunk 2 above)
Comment (adriantam) [reply tests/check/check_ttu.go:157]: Added both empty_userset and empty_direct

**Review by miparnisari — APPROVED** [review]

**Pull request merged**